### PR TITLE
Update docker images

### LINKED ISSUE #103: Update travis on root folder to do sample config and start the stack using installRadarStack
Update travis on root folder to do sample config and start the stack using installRadarStack. This would help to ensure that future releases are stable

### PR DESCRIPTION
**Pull request by nivemaham**
- updated radar-images with newer images
- adds scripts to run `install-radar-stack.sh` `instead of docker-compose up`
Fixes #103

### CODE REVIEW

File: dcompose-stack/radar-cp-hadoop-stack/migrate-postgres-data.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+echo "Stopping postgres..."
+docker-compose stop managementportal-postgresql
+docker-compose rm -vf managementportal-postgresql
+
+echo "Moving dumped data to new volume"
+mv "${MP_POSTGRES_DIR}/data/" "${MP_POSTGRES_DIR}/data-backup-$(date +%FT%TZ)/"
+mv "${POSTGRES_NEW_DIR}" "${MP_POSTGRES_DIR}/data/"
+
+# change postgres version
+sed -i "s| image: postgres:.*| image: postgres:${POSTGRES_NEW_VERSION}|" docker-compose.yml
```
Comment (blootsvoets) [inline dcompose-stack/radar-cp-hadoop-stack/migrate-postgres-data.sh]: Replace `.env` version instead

**Comment by yatharthranjan** [comment]
Can you please also update the connectors to latest versions. Thanks

File: dcompose-stack/radar-cp-hadoop-stack/migrate-postgres-data.sh (hunk 1 above)
Comment (blootsvoets) [reply dcompose-stack/radar-cp-hadoop-stack/migrate-postgres-data.sh]: use `util.sh` `sed_i` for macOS compatibility

**Comment by nivemaham** [comment]
We are getting build failure since we have ` image: postgres:${POSTGRES_NEW_VERSION}` in current version of `docker-compose` and there is no `.env` file available with the value for the environment variable. We only have `env.template`. 

Would adding a temporary `.env` just with `POSTGRES_NEW_VERSION` make an acceptable solution? @yatharthranjan @blootsvoets . 
Any comments on how we can approach this?

**Comment by yatharthranjan** [comment]
Hi nivethika. You need compose version 1.15+ for this to work.

**Comment by nivemaham** [comment]
still doesn't work. how about `cp etc/env.template .env` on travis?

File: .travis.yml [hunk 2]
```diff
@@ -39,6 +39,8 @@ script:
   - cp etc/webserver/nginx.conf.template etc/webserver/nginx.conf
   - cp etc/hdfs-connector/sink-hdfs.properties.template etc/hdfs-connector/sink-hdfs.properties
   - cp etc/mongodb-connector/sink-mongo.properties.template etc/mongodb-connector/sink-mongo.properties
+  - . etc/env.template
```
Comment (yatharthranjan) [inline .travis.yml]: Use `set -a` above this line so it exports to the environment of subsequent commands

Comment (yatharthranjan) [reply .travis.yml]: Yes cp seems fine too but the set -a should work too. But yes cp seems good since its a step to fixing #103

**Comment by nivemaham** [comment]
I have done some changes to the `travis.yml` and added mock configs for `travis build` to ensure the `install-radar-stack.sh` works upon changes. Can you guys have a look? @yatharthranjan @blootsvoets

File: dcompose-stack/radar-cp-hadoop-stack/travis/setup-env.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+echo "Starting to configure mock configurations for test"
+# create hadoop netword
+sudo docker network create hadoop
+
+# create volumes
+sudo docker volume create certs
+sudo docker volume create certs-data
```
Comment (yatharthranjan) [inline dcompose-stack/radar-cp-hadoop-stack/travis/setup-env.sh]: The networks and volumes should be created by install-radar-stack.sh script, right ?

**Pull request merged**